metrics/coverage_high_water_mark

```diff
@@ -1 +1 @@
-93.6900
\ No newline at end of file
+93.6600
```

Comment:
There's a few more code twists now to do the lazy importing which were test driven by the integration tests (which don't count towards this coverage number).  Asking for two dev approval.